msal/src/main/java/com/microsoft/identity/client/IPublicClientApplication.java

```diff
@@ -189,21 +191,21 @@ void acquireToken(@NonNull final Activity activity,
          * @param userCode user code
          * @param message instruction message
          */
-        void onUserCodeReceived(@NonNull String vUri, @NonNull String userCode, @NonNull String message);
+        void onUserCodeReceived(String vUri, String userCode, String message);
```

Comment:
nit: why removing NonNull?

Reply:
here and below

Reply:
I think it was because this was inconsistent with the rest of the code, but I went ahead and added `NonNull` to the rest of the callback